src/main/java/io/kestra/plugin/stripe/customer/Create.java

```diff
@@ -0,0 +1,123 @@
+package io.kestra.plugin.stripe.customer;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.stripe.exception.StripeException;
+import com.stripe.model.Customer;
+import com.stripe.param.CustomerCreateParams;
+import io.kestra.core.models.annotations.Example;
+import io.kestra.core.models.annotations.Plugin;
+import io.kestra.core.models.annotations.PluginProperty;
+import io.kestra.core.models.property.Property;
+import io.kestra.core.models.tasks.RunnableTask;
+import io.kestra.core.runners.RunContext;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.kestra.plugin.stripe.AbstractStripe;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Schema(
+    title = "Create a new customer in Stripe.",
+    description = "This task creates a customer in Stripe with optional name, email, and metadata."
+)
+@Plugin(
+    examples = {
+        @Example(
+            title = "Create a customer with name, email, and metadata",
+            full = true,
+            code = """
+                id: create_customer
+                namespace: company.team
+
+                tasks:
+                  - id: create_customer
+                    type: io.kestra.plugin.stripe.customer.Create
+                    apiKey: "{{ secret('STRIPE_API_KEY') }}"
+                    name: "John Doe"
+                    email: "john@example.com"
+                    metadata:
+                      plan: "pro"
+                      signup_source: "landing_page"
+                """
+        )
+    }
+)
+public class Create extends AbstractStripe implements RunnableTask<Create.Output> {
+
+    @Schema(title = "The customer's name.")
+    private Property<String> name;
+
+    @Schema(title = "The customer's email address.")
+    private Property<String> email;
```

Comment:
Shouldn't those 2 properties be `@NotNull`?

Reply:
done